src/components/Switch/Switch.tsx

```diff
@@ -1,30 +1,31 @@
-import React from "react"
-import {Switch as SwitchAria} from 'react-aria-components';
+import React from "react";
+import { Switch as SwitchAria } from "react-aria-components";
 import { base, indicator } from "./Switch.css";
 
-
 type Props = {
-  size?: 'normal' | 'small'
-  disabled?: boolean
-  onChange?: VoidFunction
-  isSelected?: boolean
-}
-
-
+  disabled?: boolean;
+  isSelected?: boolean;
+  onChange?: VoidFunction;
+  size?: "normal" | "small";
+};
 
-export const Switch:React.FC<Props> =
-  ({size = 'normal', onChange, isSelected, disabled}) => {
+export const Switch: React.FC<Props> = ({
+  size = "normal",
+  onChange,
+  isSelected,
+  disabled,
+}) => {
   const optionalProps = {
-    ...(onChange ? {onChange: onChange}: {}),
-    ...(typeof isSelected !== 'undefined' ? {isSelected: isSelected}: {})
-  }
+    ...(onChange ? { onChange: onChange } : {}),
+    ...(typeof isSelected !== "undefined" ? { isSelected: isSelected } : {}),
```

Comment:
Maybe i don't understand, but why we need this check instead set as `false` like default?

Reply:
ask @pietrograndi , I've just fixed the linting issues 😄